src/commands/CountryCommand.ts

```diff
@@ -0,0 +1,385 @@
+import { Command } from "../Command";
+import { CommandInteraction, MessageEmbed } from "discord.js";
+import {
+	inlineCode,
+	SlashCommandBuilder,
+	SlashCommandStringOption,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "@discordjs/builders";
+import {
+	Country,
+	countryData,
+	CountryKey,
+	getCountryByName,
+	getFilteredCountryDataBy,
+	initializeCountryData,
+	sortCountryDataBy,
+	typeOfCountryProperty,
+} from "../util/countryDataManager";
+import { formatNumber } from "../util/numbers"
+
+let locale: string = "";
+
+class CountryCommand extends Command {
+	constructor() {
+		super("country");
+		initializeCountryData();
+	}
+
+	async execute(interaction: CommandInteraction): Promise<void> {
+		locale = interaction.locale;
```

Comment:
is reset every execution anyways, and `interaction` is also accessible in every sub function no need to define it outside the execute method

Reply:
but the functions that use it are not part of the class and I don't want to pass it to them all the time